### PR TITLE
add css-rules for thumbnails in listings and portlets (related to htt…

### PR DESCRIPTION
**Pull request by fgrcon**
…ps://github.com/plone/Products.CMFPlone/issues/1226)

### CODE REVIEW

File: plonetheme/barceloneta/theme/less/contents.plone.less [hunk 1]
```diff
@@ -37,4 +37,12 @@ body#visual-portal-wrapper.pat-plone #edit-zone {
...
 	.contenttype-news-item:after {	content: '\e80f';}
-}
\ No newline at end of file
+}
+
+//*// thumbnails for images, leadimages  in listings and portlets
+    .image-icon, .image-tile {
+      float:right;
+}
+// to supress thumbnails in listings and/or portlets uncomment the following
```
Comment (gforcada) [inline plonetheme/barceloneta/theme/less/contents.plone.less]: That's probably better kept as documentation ;-)

@polyester @svx any hints where to put this?

Comment (thet) [inline plonetheme/barceloneta/theme/less/contents.plone.less]: In docs additionally to this comment here... I like CSS comments like this here. Normally CSS files are heavily underdocumented.

Comment (polyester) [inline plonetheme/barceloneta/theme/less/contents.plone.less]: +1 on inline documentation in CSS

Comment (davilima6) [inline plonetheme/barceloneta/theme/less/contents.plone.less]: +1 too

**Pull request merged**